.eslintrc

```diff
@@ -5,6 +5,7 @@
   ],
   "plugins": [
     "prettier",
+    "react",
```

Comment:
No need to use this as `airbnb` already imports it

Reply:
I thought it was weird that plugin wasn't being used. Why the hell we need to have it as a separate dependency instead of havinb `eslint-config-airbnb` install it has its own dependency beats me. Fixed in 090ff8e.